book/docs/Open_formats_tools.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<p>I would link to <a href="https://stacspec.org/en" target="_blank">https://stacspec.org/en</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>